bin/handler-opsgenie.rb

```diff
@@ -58,6 +62,7 @@ def create_alert(description)
     tags << 'unknown' if event_status >= 3
     tags << 'critical' if event_status == 2
     tags << 'warning' if event_status == 1
+    event_tags.each { |tag, value| tags << "#{tag}_#{value}" }
```

Comment:
Your right,

I test with and without tags, and the `.try(:each)` doesn't work. I directly verified if the class is nil or not with `unless` keyword